<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particle Editor Alpha</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="Options">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>271</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLineEdit" name="lineMinScale">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Smallest scale of the particles</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMaxScale">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Biggest scale of particles</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ApplyAll">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>460</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinScale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Min Scale</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxScale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Scale</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboShape">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Box</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sphere</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelShape">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Shape</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PickBrightest">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change brightest</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PickDarkest">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change darkest</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupDirection">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Direction</string>
     </property>
     <widget class="QLineEdit" name="dirX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirY">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.05</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirZ">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDirection">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X              Y                Z</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelDuration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineDuration">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Biggest scale of particles</string>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMinRate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Biggest scale of particles</string>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinRate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Min Rate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMaxRate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Biggest scale of particles</string>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxRate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Rate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMinTime">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Biggest scale of particles</string>
     </property>
     <property name="text">
      <string>500</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Min Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Time</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMaxTime">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Biggest scale of particles</string>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Box settings</string>
     </property>
     <widget class="QLabel" name="labelMinBoxDimension">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Min dimensions</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMaxBoxDimension">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Max dimensions</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMinBoxX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMinBoxY">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMinBoxZ">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMaxBoxZ">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMaxBoxY">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineMaxBoxX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupSphereSettings">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Sphere settings</string>
     </property>
     <widget class="QLineEdit" name="lineSpereRadius">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSphereRadius">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="ParticlePreviewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJan">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuJan"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
